appveyor.yml

```diff
@@ -17,6 +18,6 @@ test: off
 
 artifacts:
     - path: .\artifacts\logs\*.*
-    - path: .\artifacts\output\*-netstd.nupkg
+    - path: .\artifacts\output\*-netstd*.nupkg

```

Comment:
The build artifacts looks good now: https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/build/18/artifacts